### DISCUSSION TITLE
solution to no response macros: Signing own macros with SelfCert.exe

### DISCUSSION THREAD

**Discussion by mbkom** [tallguyjenks/Obsidian-For-Business#13] [General]
Had problems getting the macros to work even with settings in Trust Centre. Went into settings and fixed it by signing macros with SelfCert.exe [(guide to find and use SelfCert.exe)](https://www.howto-outlook.com/howto/selfcert.htm#locating). 

After SelfCert and restart of Outlook the 'Quick Access' macro buttons started responding.

**Comment by tallguyjenks** [comment]
interesting workaround, Strange that it was necessary though
